### PR TITLE
UX Improvements

### PR DESCRIPTION
**Pull request by PseudoMera**

### CODE REVIEW

File: src/components/modals/private-key/revealPrivateKey.jsx [hunk 1]
```diff
@@ -50,6 +58,14 @@ export default function RevealPrivateKey({ visible, onClose, ppk }) {
     setPassphrase(value);
   }, []);
 
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setDisplayAlert(false);
+    }, 500);
```
Comment (Evalir) [inline src/components/modals/private-key/revealPrivateKey.jsx:64]: Are we sure we only wanna display the alert for 500ms?

File: src/views/create/createWallet.jsx [hunk 2]
```diff
@@ -36,7 +37,7 @@ function Create({
 
         if (!isPassphraseValid(value)) {
           setPassPhraseError(
-            "Passphrase must be minimum 15 characters, 1 min uppercase letter and 1 special character."
+            "Passphrase must be minimum 8 characters, 1 min uppercase letter and 1 special character."
```
Comment (Evalir) [inline src/views/create/createWallet.jsx:40]: Q: Where are the changes related to this change in length for passphrases? Did we do them in an earlier commit?

**Review by Evalir — APPROVED** [review]
LGTM! Just a few questions

File: src/views/create/createWallet.jsx (hunk 2 above)
Comment (PseudoMera) [reply src/views/create/createWallet.jsx:40]: No, it has always worked that way.

**Pull request merged**